src/components/TypeTwo/Blueprint.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
```

Comment:
Made this a .js for now in case I want somewhere to mock things in JS later with appropriate syntax highlighting.

Reply:
I could also add it as an issue or to the wiki, as not to clutter the codebase, but I think it's good to have around until the transition is complete.